arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
+        'display': 'inline-block !important',
+        '-moz-transform': 'scaleX(-1)',
+        '-webkit-transform': 'scaleX(-1)',
+        '-o-transform': 'scaleX(-1)',
+        'transform': 'scaleX(-1)',
+        '-ms-filter': 'fliph',
+        'filter': 'fliph'
+      },
+      '.mar': {
+        'font-family': 'Amiri !important',
+        'font-style': 'normal !important'
+      }
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('mml Jax Ready', function () {
+  MathJax.Hub.Register.StartupHook('HTML-CSS Jax Ready', function () {
+    MathJax.Hub.Register.StartupHook('Arabic TeX Ready', function () {
```

Comment:
@dpvc I still like the idea of splitting the extension into files ([in dev. repo only!](https://github.com/Edraak/arabic-mathjax-dev)). So the `Arabic TeX Startup`, so I'm keeping the signal. 

I've also added a final `Arabic Ready` signal to enable future extension.